src/library.js

```diff
@@ -13,24 +12,39 @@ import resolve from "./resolve.js";
 import requirer from "./require.js";
 import svg from "./svg.js";
 import tex from "./tex.js";
+import vegalite from "./vegalite.js";
 import width from "./width.js";
 
 export default Object.assign(function Library(resolver) {
   const require = requirer(resolver);
-  Object.defineProperties(this, {
-    DOM: {value: DOM, writable: true, enumerable: true},
-    FileAttachment: {value: constant(NoFileAttachments), writable: true, enumerable: true},
-    Files: {value: Files, writable: true, enumerable: true},
-    Generators: {value: Generators, writable: true, enumerable: true},
-    html: {value: constant(html), writable: true, enumerable: true},
-    md: {value: md(require), writable: true, enumerable: true},
-    Mutable: {value: constant(Mutable), writable: true, enumerable: true},
-    now: {value: now, writable: true, enumerable: true},
-    Promises: {value: Promises, writable: true, enumerable: true},
-    require: {value: constant(require), writable: true, enumerable: true},
-    resolve: {value: constant(resolve), writable: true, enumerable: true},
-    svg: {value: constant(svg), writable: true, enumerable: true},
-    tex: {value: tex(require), writable: true, enumerable: true},
-    width: {value: width, writable: true, enumerable: true}
-  });
+  Object.defineProperties(this, properties({
+    DOM: () => DOM,
+    FileAttachment: () => NoFileAttachments,
+    Files: () => Files,
+    Generators: () => Generators,
+    Inputs: () => require("@observablehq/inputs@0.7.21/dist/inputs.umd.min.js"),
+    Mutable: () => Mutable,
+    Plot: () => require("@observablehq/plot@0.1.0/dist/plot.umd.min.js"),
+    Promises: () => Promises,
+    _: () => require("lodash@4.17.21/lodash.min.js"),
```

Comment:
This one is sort of not-like-the-others in this set. I assume that it is very commonly used and stable in notebooks?

Reply:
Lodash is the most popular library on npm according to this analysis:

https://gist.github.com/anvaka/8e8fa57c7ee1350e3491

It appears to be referenced in 2.5K public notebooks according to this query:

https://observablehq.com/search?query=lodash

That’s less than Vega (9K) and D3 (10K+), but still quite popular, and I vouch for its quality.

Reply:
Question regarding evolvability and compatibility: I started out using Ramda because of its elegant “immutable auto-curried iteratee-first data-last methods”. However, including the much more popular `lodash` library conveniently and lazily by default urges me to switch to `lodash`. Plus, I just learned that there also is `lodash/fp`, that enjoys the same feature as Ramda above. Is there a way to include `lodash/fp` by default the same way as just `lodash`? Or should I fall back to an explicit require?

- https://github.com/lodash/lodash/wiki/FP-Guide
- https://observablehq.com/@mbostock/hello-lodash-fp